sql/src/main/java/io/crate/metadata/shard/unassigned/UnassignedShard.java

```diff
@@ -79,38 +76,25 @@ public static int markAssigned(int shard) {
     private final int id;
     private final String partitionIdent;
     private final BytesRef state;
+    private final boolean orphanedPartition;
 
     private static final BytesRef UNASSIGNED = new BytesRef("UNASSIGNED");
     private static final BytesRef INITIALIZING = new BytesRef("INITIALIZING");
 
-    private Boolean orphanedPartition = false;
 
-    public UnassignedShard(ShardId shardId,
+    public UnassignedShard(int shardId,
+                           String indexName,
                            ClusterService clusterService,
                            Boolean primary,
                            ShardRoutingState state) {
-        String index = shardId.getIndexName();
-        boolean isBlobIndex = BlobIndex.isBlobIndex(index);
-        final String schemaName, tableName, partitionIdent;
-        if (isBlobIndex) {
```

Comment:
I guess this is not need because of a recent change regarding the schema for blob tables?

Reply:
Became obsolete with b75b20964a